comp/volunteers/refreshments.yaml

```diff
@@ -1,6 +1,6 @@
-summary: Plan refreshments for volunteers
+summary: Plan lunch & refreshments for volunteers
```

Comment:
Lunch feels sufficiently separate from refreshments to warrant its own ticket. Refreshments is definitely a requirement, but lunch is more of a "should" IMO

Reply:
Makes sense for someone to take on organising both here. It's easier for the task to be split up if needed rather that keeping track of merging tasks.

Reply:
I'm mixed on this. I generally agree that organising food & refreshments in general feels like it's one area, but I'm not convinced by some of the logic in the description as to the suggested approach.

> volunteers often don't have time to go out for lunch

This is not really true. We do have a lunch break in the schedule and in the past volunteers have in the past often had plenty of time to go to food vendors to get food and have a break. We have been less good at this in recent years, though IMO this reflects on how we've approached the scheduling and the level of things which needed addressing over lunch rather than a hard limitation.

> we should organise communal food for lunch during the competition

I don't believe this follows as a statement even if the previous suggestion (that volunteers don't have time to go very far) were true. An alternative approach would be to ensure there were food vendors on site or very near by. There is absolutely not a requirement that the food be provided by SR nor that it be communal.

---

In sharp contrast to the above, while on station, volunteers in many roles literally cannot go and get their own refreshments. This could be for safety or other reasons associated with their role. Provision of refreshments, _taken to_ these volunteers is therefore very important to avoid them going without a drink for whole days at a time. This sounds extreme, but did happen in the early days of SR and that was the origin of the tickets around refreshments.

I think it's important that the different levels of importance of these requirements are reflected clearly in the recurring tasks.

Reply:
We've clarified the wording of this slightly ([d5392c9](https://github.com/srobo/recurring-tasks/pull/90/commits/d5392c963b2f5ab883160904f0ddbab8a601237f)).

I'd argue that having provisions/a plan for lunch and ensuring those in safety critical roles are fed/watered are both a must have and do not differ in priority.